ISISComputingGroup/ibex_gui
# Ticket 4081: Very unresponsive jaws

### Description of work

Moved jaws calculations into one script and off of the GUI thread.

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/3764

To test:
* On master create a jawset and reflectometry sever (see https://github.com/ISISComputingGroup/EPICS-inst_servers/pull/191)
* Open a jaws OPI and reflectometry OPI
* Start a move of the jaws
* Switch between OPIs, notice that the GUI slows right down and stops getting updates from the motors as quickly
* Do the same on this branch, hopefully the speed will be much improved
* Confirm that the jaws diagram still works as expected

### Documentation

Added notes about threading in:
* https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/OPI-programming-tips-and-limitations
* https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/OPI-Creation

---

#### Code Review


### Final Steps
